docs/ja/reference/path.md

```
@@ -0,0 +1,86 @@
+[English](../../reference/path.md) | [日本語](path.md) | [繁體中文](../../zh/reference/path.md)
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)